user_docs/en/userGuide.t2t

```diff
@@ -1115,7 +1115,7 @@ For no pitch change you would use 0.
 
 ==== Say "cap" before capitals ====[SpeechSettingsSayCapBefore]
 This setting is a checkbox that, when checked, tells NVDA to say the word "cap" before any capital letter when spoken as an individual character such as when spelling.
-Usually, NVDA raises the pitch slightly for any capital letter, but some synthesizers may not support this well, so perhaps this option may be of use.
+Usually, NVDA raises the pitch slightly for any capital letter, but some synthesizers may not support this well, so perhaps this option may be preferred alternative for some users.

```

Comment:
On line 1122 (under "Beep for capitals")

Instead of:
> Like the "say cap for capitals" checkbox, this is useful for Synthesizers that can't change their pitch for capital letters.

Use the same sentence:
> This option is also useful for Synthesizers that can't change their pitch for capital letters.